src/python/pants/backend/native/subsystems/native_toolchain.py

```diff
@@ -32,6 +32,15 @@ class NativeToolchain(Subsystem):
 
   options_scope = 'native-toolchain'
 
+  @classmethod
+  def register_options(cls, register):
+    super(NativeToolchain, cls).register_options(register)
+
+    register('--enable-libc-search', type=bool, default=False, fingerprint=True, advanced=True,
```

Comment:
It seems like maybe this should be defined next to `--libc-dir`, and be mutually exclusive?

Reply:
Hm. But then, this flag wouldn't make sense at all on OSX, so it would be weird to have it on the cross-platform subsystem. I'll move it to the libc subsystem.